test/integration/install.test.js

```diff
@@ -67,9 +67,15 @@ describe('snowpack install', () => {
         removeLockfile(testName);
       }
 
+      const cwd = path.join(__dirname, testName);
+      if (!existsSync(path.join(cwd, 'package.json'))) {
+        console.error(testName, 'no longer exists, skipping...');
+        return;
```

Comment:
Beware! This is a `for()` loop! Which means you must `continue` instead.

_—Signed, someone that broke our entire test suite by doing this 😅 _

Reply:
Also this can be a `forEach()` if that’s safer; it doesn’t change anything

Reply:
It's in a loop, but wrapped by the `it()` function, so the `return` is fine here, or am I missing something?

Reply:
Oh—you’re right. We’re fine here. Carry on.

Reply:
not the first time we've run into this trickiness tho! At the very least, a good reminder to be careful when messing with the control flow of the test runner.